### PR TITLE
feat: update layout

### PR DESCRIPTION
**Pull request by ericzakariasson**
- add layout component
- update landing page
- update publication layout
- add skeleton loading

### CODE REVIEW

**Comment by Kenny-Gin1** [comment]
Clean and beautiful! 
image

**Review by Kenny-Gin1 — APPROVED** [review]

File: apps/web/src/components/PageEditor.tsx [hunk 1]
```diff
@@ -88,33 +77,35 @@ export function PageEditor({ page, onSave, isSaving }: PageEditorProps) {
...
-          )}
-          {isSaving ? "Saving..." : "Save page"}
-        </Button>
-      </div>
+      {renderSubmit({
+        isDisabled: !isEnabled,
+        data: {
+          page: {
+            title,
+            content: json?.content ?? [],
```
Comment (philipandersson) [inline apps/web/src/components/PageEditor.tsx:103]: At some point we need to get type validation in 😄 
zod to the rescue

File: apps/web/src/pages/[document].tsx [hunk 2]
```diff
@@ -55,22 +57,26 @@ export const getServerSideProps: GetServerSideProps<Props> = async (ctx) => {
 const DocumentPage: NextPage<Props> = ({ page: initialPage }) => {
   const [page, setPage] = useState<DeserializedPage | null>(null);
   const [isEditing, setIsEditing] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
 
   const { address } = useAccount();
 
   const handlePageLoad = useCallback(async () => {
+    setIsLoading(true);
```
Comment (philipandersson) [inline apps/web/src/pages/[document].tsx:65]: I guess this could have been a `react-query` query?

**Review by philipandersson — APPROVED** [review]

File: apps/web/src/pages/[document].tsx (hunk 2 above)
Comment (ericzakariasson) [reply apps/web/src/pages/[document].tsx]: Added now with optimistic updates 🚀

**Pull request merged**